shell.nix

```diff
@@ -23,9 +23,14 @@ let
     (pkgs.writeScriptBin "install-editor" ''
       #!/usr/bin/env bash
       set -e
-      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/utopia-vscode-common
+      update-vscode-build-extension
```

Comment:
in https://github.com/concrete-utopia/utopia/pull/1099/files#diff-e53dfbfffe62ae3c0b411b3938ccffa9fb6a2ecc565f55785ef8daa756631a6bL270 

didn't you rename `update-vscode-build-extension` to `pull-extension` ?